docs/Deploying-Pega-on-AKS.md

```diff
@@ -421,17 +421,11 @@ To configure the parameters in the backingservices.yaml file, download the file
+| srs.srsStorage.provisionInternalESCluster: | Enabled by default to provision an Elasticsearch cluster | <ul><li>srs.srsStorage.provisionInternalESCluster:`true` download `Makefile` from the [repository](https://github.com/pegasystems/pega-helm-charts) ( path from root : `pega-helm-charts/charts/backingservices/Makefile`) and replace `<YOUR_NAMESPACE> ` with `namespace` used for the deployment run `$ es-prerequisite `</li><li>srs.srsStorage.provisionInternalESCluster:`false` if you want to use an existing, externally provisioned ElasticSearch cluster. </li></ul> |
 | srs.srsStorage.domain: port: protocol: basicAuthentication: awsIAM: requireInternetAccess: | Disabled by default. Enable only when srs.srsStorage.provisionInternalESCluster is false and you want to configure SRS to use an existing, externally provisioned Elasticsearch cluster. For an Elasticsearch cluster secured with Basic Authentication, use `srs.srsStorage.basicAuthentication` section to provide access credentials. For an AWS Elasticsearch cluster secured with IAM role based authentication, use `srs.srsStorage.awsIAM` section to set the aws region where AWS Elasticsearch cluster is hosted. For unsecured managed ElasticSearch cluster do not configure these options. | <ul><li>srs.srsStorage.domain: "\<external-es domain name\>"</li> <li>srs.srsStorage.port: "\<external es port\>"</li> <li>srs.srsStorage.protocol: "\<external es http protocol, `http` or `https`\>"</li>     <li>srs.srsStorage.basicAuthentication.username: "\<external es `basic Authentication username`\>"</li>     <li>srs.srsStorage.basicAuthentication.password: "\<external es `basic Authentication password`\>"</li>     <li>srs.srsStorage.awsIAM.region: "\<external AWS es cluster hosted `region`\>"</li><li> srs.srsStorage.requireInternetAccess: "\<set to `true` if you host your external Elasticsearch cluster outside of the current network and the deployment must access it over the internet.\>"</li></ul>     |
 | elasticsearch: volumeClaimTemplate: resources: requests: storage: | Specify the Elasticsearch cluster disk volume size. Default is 30Gi, set this value to at least three times the size of your estimated search data size | <ul><li>elasticsearch: volumeClaimTemplate: resources: requests: storage:  "\<30Gi>” </li></ul> |
 
 3. Save the file.
-4. In case of using internal Elasticsearch cluster (`srs.srsStorage.requireInternetAccess:true`) please download `Makefile` from the [repository](https://github.com/pegasystems/pega-helm-charts) ( path from root : `pega-helm-charts/charts/backingservices/Makefile`) and replace `<YOUR_NAMESPACE> ` with `namespace` used for the deployment run
```

Comment:
Update step 4 to say>>
4. To use an internal Elasticsearch cluster (`srs.srsStorage.requireInternetAccess:true`) for your deployment, you must download the `Makefile` file from the [repository](https://github.com/pegasystems/pega-helm-charts) ( path from root : `pega-helm-charts/charts/backingservices/Makefile`) and replace `<YOUR_NAMESPACE> ` with the `namespace` you used for the deployment, enter one of the following commands:

Reply:
done.